doc/source/user_guide/ug_case_settings.rst

```diff
@@ -249,7 +249,9 @@ grid_nml
    "", "``pop``", "pop thickness file in cm in ascii format", ""
    "``close_boundaries``", "logical", "force two gridcell wide land mask on boundaries for rectangular grids", "``.false.``"
    "``dxrect``", "real", "x-direction grid spacing for rectangular grid in cm", "0.0"
+   "``dxscale``", "real", "user defined rectgrid x-grid scale factor", "1.0"
    "``dyrect``", "real", "y-direction grid spacing for rectangular grid in cm", "0.0"
+   "``dyscale``", "real", "user defined rectgrid y-grid scale factor", "1.0"
```

Comment:
Is the variable rectgrid scaling not described elsewhere in the docs?  I don't see it.  Would be good to add a brief description somewhere, maybe in one of the sections that mention the box tests like 3.3.1.3.1. box2001.

Reply:
Agree this should be addressed on Consortium main and should have been addressed better when the variable grid stuff was added.  I have added an issue so we don't forget, https://github.com/CICE-Consortium/CICE/issues/770.